docs/resources/ec2-image.md

```diff
@@ -0,0 +1,36 @@
+# EC2 Image
+
+This will remove all IAM Roles an AWS account.
```

Comment:
This will remove all "EC2 Images (AMI)" in an AWS account

Reply:
Great catch.